src/web3py/extensions/contracts.py

```diff
@@ -0,0 +1,88 @@
+import json
+from functools import lru_cache
+
+from web3 import Web3
+from web3.module import Module
+from web3.types import Wei
+
+from src import variables
+from src.typings import BlockStamp
+
+
+class LidoContracts(Module):
+    def __init__(self, w3: Web3):
+        super().__init__(w3)
+        self._load_contracts()
+
+    def _load_contracts(self):
+        # Contract that stores all lido contract addresses
```

Comment:
let's print addresses of all contracts

Reply:
Don't cache it on init: always take it from LidoLocator.
The same for ejector module
And beware the same in other places

Reply:
https://www.notion.so/ebbe9321e1544043a04679d991a4b3ec?v=fae8607f93fc4299b1fdf0182142fb5c&p=7fc96ea121294b10b963a47d539f8c26&pm=s